corehq/ex-submodules/casexml/apps/case/tests/test_multimedia.py

```diff
@@ -75,19 +77,24 @@ def _calc_file_hash(self, key):
         with open(MEDIA_FILES[key], 'rb') as attach:
             return hashlib.md5(attach.read()).hexdigest()
 
-    def _do_submit(self, xml_data, dict_attachments):
+    def _do_submit(self, xml_data, dict_attachments, sync_token=None):
         """
         RequestFactory submitter - simulates direct submission to server directly (no need to call process case after fact)
         """
-        form = post_xform_to_couch(xml_data, dict_attachments)
-        form.domain = TEST_DOMAIN
+        sp = couchforms.SubmissionPost(
+            instance=xml_data,
+            domain=TEST_DOMAIN,
+            attachments=dict_attachments,
+            last_sync_token=sync_token,
+        )
+        response, xform, cases = sp.run()
```

Comment:
prior to this change the multimedia tests weren't exercising the atomic submissions code